test/integration/docker-compose-localbuild.yml

```diff
@@ -0,0 +1,12 @@
+# this YAML file overrides settings in the cbioportal-docker-compose YAML
+# and maps a local maven build to appropriate locations in the container (which is retrieved by image).
+
+version: '3'
+
+services:
+  cbioportal:
+    command: /bin/sh -c "java -Xms2g -Xmx4g -Dauthenticate=noauthsessionservice -Dsession.service.url=http://cbioportal_session:5000/api/sessions/my_portal/ -jar webapp-runner.jar -AmaxHttpHeaderSize=16384 -AconnectionTimeout=20000 --enable-compression /app.war"
```

Comment:
is there a way to not override the command? Or does one always need to override the entire service?

Reply:
Well, the command is overridden since we run the WAR file itself, not the decompressed version. It removes the need for unzipping. The command does not have to be overridden for technical reasons, but I think that running the WAR directly is cleaner since it removes the need for a unzip in the travis.yml file. It is not necessary and may confuse people in the future. Remember that the only reason for running a unzipped version is that this allows to mount custom icons in an instance.

Reply:
we run the WAR file to override running the files in the cbioportal image

Reply:
I see makes sense